components/raftstore/src/store/fsm/store.rs

```diff
@@ -623,6 +623,9 @@ impl<'a, EK: KvEngine + 'static, ER: RaftEngine + 'static, T: Transport>
                     inspector.record_store_wait(send_time.saturating_elapsed());
                     self.ctx.pending_latency_inspect.push(inspector);
                 }
+                StoreMsg::SendDetailedReportForUnsafeRecovery => {
```

Comment:
no need this msg, you can make `store_heartbeat_pd` to be a static function and call it in peer_fsm directly, see https://github.com/tikv/tikv/pull/11921/files#diff-51096b677ec8dac18619cd087727985b354be3e529ffdcf5b32429a32b77caa3R1653

Reply:
Done.